mocks/service/launchpad/index.js

```diff
@@ -44,10 +44,7 @@ router.get('/devel/*snaps', (req, res) => {
 });
 
 router.post('/~:owner/+snap/:name', (req, res) => {
```

Comment:
Does this work for you (I mean using `+` in path)?
I had issues yesterday with trying to mock `/devel/+snaps` because router treasts `+` as  in regexp. (See: https://github.com/cjwatson/build.snapcraft.io/blob/9d4087eaf03a71c89f7c5ff59079c9e2d6457987/mocks/service/launchpad/index.js#L18-L20)